lib/workers/repository/dependency-dashboard.ts

```diff
@@ -453,3 +458,94 @@ function getFooter(config: RenovateConfig): string {
 
   return footer;
 }
+
+export async function getDashboardMarkdownVulnerabilities(
+  config: RenovateConfig,
+  packageFiles: Record<string, PackageFile[]>
+): Promise<string> {
+  let result = '';
+
+  if (
+    is.nullOrUndefined(config.dependencyDashboardOSVVulnerabilitySummary) ||
+    config.dependencyDashboardOSVVulnerabilitySummary === 'none'
+  ) {
+    return result;
+  }
+
+  result += '## Vulnerabilities\n\n';
+
+  const vulnerabilityFetcher = await Vulnerabilities.create();
+  const vulnerabilities = await vulnerabilityFetcher.fetchVulnerabilities(
+    config,
+    packageFiles
+  );
+
+  if (vulnerabilities.length === 0) {
+    result += 'None detected.\n\n';
```

Comment:
```suggestion
    result += 'No known CVEs found.\n\n';
```

Do we mean CVE or more general vulnerabilities. The dependencies may have no CVEs but the repository code could still have bad input sanitation for example.

We should be clear on _what_ type of vulnerability Renovate detects.